### PR TITLE
DDF-3945 Permission entry for the gmd:MD action

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Addresses access control exception when clicking the gmd:MD_Metadata action in the Inspector in Intrigue.
#### Who is reviewing it? 
@peterhuffer @Kjames5269 @bdeining 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@rzwiefel 
@stustison 

#### How should this be tested?
1) Ingest a file/metacard
2) Go into the Inspector tab in Intrigue
3) Go to the Action tab
4) Select the gmd:md_metadata action

#### What are the relevant tickets?
[DDF-3945](https://codice.atlassian.net/browse/DDF-3945)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/3512)

### CODE REVIEW

**Comment by shaundmorris** [comment]
build now

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 1]
```diff
@@ -121,7 +121,7 @@ grant codeBase "file:/org.apache.servicemix.bundles.wsdl4j" {
     permission java.io.FilePermission "<<ALL FILES>>", "read";
 }
 
-grant codeBase "file:/org.apache.karaf.shell.core/registry-federation-admin-service-impl/catalog-core-commands/spatial-geocoding-offline-catalog/spatial-geocoding-geocoder/spatial-geocoding-plugin/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source/spatial-wfs-featuretransformer/spatial-wfs-featuretransformer-xstream/resourcemanagement-usage-ui" {
+grant codeBase "file:/org.apache.karaf.shell.core/registry-federation-admin-service-impl/catalog-core-commands/spatial-geocoding-offline-catalog/spatial-geocoding-geocoder/spatial-geocoding-plugin/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source/spatial-wfs-featuretransformer/spatial-wfs-featuretransformer-xstream/resourcemanagement-usage-ui/catalog-rest-endpoint" {
```
Comment (Kjames5269) [inline distribution/ddf-common/src/main/resources/security/default.policy]: ❓was this just a name change on the bundle or was it not being specific enough? Want to clarify that the former ```resourcemanagement-usage-ui``` won't need those permissions for anything

Comment (Kjames5269) [reply distribution/ddf-common/src/main/resources/security/default.policy]: No that was my bad not understanding it fully, I thought it was a large filepath. after clarifying with @bakejeyner looks fine :+1:

**Comment by peterhuffer** [comment]
Successful hero. Ingested a product via Intrigue and verified the the `gmd:MD_Metadata` action did not throw any exceptions and gave back a valid response.

**Pull request merged**